f3-sidecar/README.md

```diff
@@ -0,0 +1,4 @@
+### EC tests
+
+- run a forest node locally and expose RPC port and the default 2345
+- run `go test -v .`
```

Comment:
Is there a specific Go version required?

Reply:
Yes Go1.21 specified in `go.mod`. Since a certain Go version, Go toolchain is capable of automatically downloading the proper compiler just like rust does